test.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+rm -rf test
+mkdir -p test
+mkdir -p test/mount_point
+mkdir -p test/gitfs_owned_repo
+mkdir -p test/origin.git
+cat > test/.gitconfig <<EOF 
+[user] 
+       name = GitFs
+       email = gitfs@gitfs.com\n
+
+EOF
+
+ROOT=`pwd`
+
+cd test/origin.git;\
+    HOME=../../test git init --bare .;\
+    cd ../../;\
```

Comment:
Agreed. I just sort of copied what drone was doing, but I'll clean it up now. Postponing cleanup  tends to mean it never gets done :)